HelpSource/Classes/HelpBrowser.schelp

```diff
@@ -7,6 +7,10 @@ description::
 
 HelpBrowser is the GUI help browser that lets you browse the documentation of SuperCollider. It is coupled with SCDoc to allow on-the-fly rendering of HTML help files.
 
+Note that this is not the same as the help browser built into the IDE.
```

Comment:
I think this is a good note to mention but along with the line below can be a bit confusing. Maybe something like adding ", although they both use the same underlying Qt WebEngine dependency."